eng/pipelines/libraries/deploy-networking-echo-test-servers.yml

```
@@ -0,0 +1,77 @@
+trigger: none
+
+resources:
+  repositories:
+    - repository: 1ESPipelineTemplates
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+
+variables:
+  - template: /eng/common/templates-official/variables/pool-providers.yml
+  - name: FrameworkVersion
+    value: 8.x
+  - name: TargetFramework
+    value: net8.0
+
+extends:
+  template: v1/1ES.Unofficial.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    customBuildTags:
+      - 1ES.PT.ViaStartRight
+    pool:
+      name: $(DncEngInternalBuildPool)
+      image: 1es-windows-2022
+      os: windows
+
+    stages:
+      - stage: stage
+        displayName: Building in a VM
+        jobs:
+          - job: job
+            displayName: Build
+            steps:
+              - checkout: self
+
+              - task: UseDotNet@2
+                inputs:
+                  packageType: "sdk"
+                  version: $(FrameworkVersion)
+
+              - script: echo {} > ./global.json
+                displayName: Bypass repository global.json
+                workingDirectory: ./src/libraries/Common/tests/System/Net/Prerequisites/NetCoreServer
+
+              - script: dotnet publish -c Release /p:GenevaTelemetry=true /p:_TargetFrameworkForXHarness=$(TargetFramework)
+                workingDirectory: ./src/libraries/Common/tests/System/Net/Prerequisites/NetCoreServer
+                displayName: Run dotnet publish
+
+              - task: zip@0
+                displayName: Zip artifacts
+                inputs:
+                  pathToZipFolder: 'artifacts\bin\NetCoreServer\Release\$(TargetFramework)\publish\'
+                  pathToZipFile: 'artifacts\bin\NetCoreServer\Release\$(TargetFramework)\publish.zip'
+
+              - task: AzureRmWebAppDeployment@4
+                displayName: Deploy to corefx-net-http11
```

```suggestion
                displayName: Deploy to corefx-net-http11-staging
```
In case we want to be consistent.

Pardon my poor understanding of AZP, but do I get it right that  that running `AzureRmWebAppDeployment@4` with `WebAppName: "corefx-net-http11"` and `SlotName: "staging"` creates the DNS entry `corefx-net-http2-staging.azurewebsites.net`?

> the deployment slots need to be swapped manually

Will this automatically change the subdomain names (removing `-staging`)?

---

This deploys to the slot that is accessible via `corefx-net-http11-staging.azurewebsites.net`. Then there is a button in Azure portal to swap the app serving the staging url with the one serving `corefx-net-http11.azurewebsites.net`.

---

I see, thanks. Would it make sense to place a comment at the top of the file describing the process, or document it in some other way?

(The `displayName` suggestion still applies.)